packages/ui/src/lobbyPlayerBanner/lobbyPlayerBanner.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+import { Title } from "../player/title";
+import { LobbyPlayer } from "@creature-chess/models";
+
+
+type Props = {
+	player: LobbyPlayer | null;
+};
+
+const LobbyPlayerBanner: React.FunctionComponent<Props> = ({ player }) => {
+	if (!player) {
+		return (
+			<div key={Math.floor(Math.random() * 1000)} className="player bot">
```

Comment:
avoid random keys https://codeburst.io/how-to-not-react-common-anti-patterns-and-gotchas-in-react-40141fe0dcd